src/components/sidenavigation/SideNavigation.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { PropsWithChildren, ReactNode } from 'react';
+import { Button, Heading } from '@navikt/ds-react';
+import Link from 'next/link';
+import { Bandage, CoApplicant, Task } from '@navikt/ds-icons';
+import { useRouter } from 'next/router';
+import cn from 'classnames';
+
+import { PreviewSykmeldtFragment } from '../../graphql/queries/react-query.generated';
+import { formatNamePossessive } from '../../utils/sykmeldtUtils';
+
+import styles from './SideNavigation.module.css';
+
+interface Props {
+    sykmeldt: PreviewSykmeldtFragment | null;
+}
+
+const SideNavigation = ({ sykmeldt, children }: Required<PropsWithChildren<Props>>): JSX.Element | null => {
+    const activePage = useActivePage();
+    if (!sykmeldt) return null;
+
+    return (
+        <div className={styles.rootContainer}>
+            <nav className={styles.menuContainer}>
+                <Heading size="medium" className={styles.heading}>
+                    {formatNamePossessive(sykmeldt, 'dokumenter')}
+                </Heading>
+                <div className={styles.buttonList}>
+                    <MenuItem
+                        href={`/sykmeldt/${sykmeldt.narmestelederId}/sykmeldinger`}
+                        icon={<Bandage />}
+                        className={cn({ [styles.active]: activePage === 'sykmeldinger' })}
+                    >
+                        Sykmeldinger
+                    </MenuItem>
+                    <MenuItem
+                        href={`/sykmeldt/${sykmeldt.narmestelederId}/soknader`}
+                        icon={<Task />}
+                        className={cn({ [styles.active]: activePage === 'soknader' })}
+                    >
+                        Søknader om sykmeldinger
+                    </MenuItem>
+                    <MenuItem href="/" icon={<CoApplicant />} className={styles.lastItem}>
+                        Dine sykmeldte
+                    </MenuItem>
+                </div>
+            </nav>
+            <section className={styles.pageContainer}>{children}</section>
+            <div className={styles.sideMenuFiller} />
```

Comment:
Er det mulig å legge stylinga av `diven` i `section`, så slepp vi å ha denne `diven`?

Reply:
Jeg er litt usikker. Disse 3 siblingsene bruker flexbox til å fordele spacen. Ser ingen lett måte å fordele det på samme måte uten å bruke denne tomme divven. :thinking: Er åpen for gode ideèr

Reply:
`margin-right` på `<section className={styles.pageContainer}>` ?

Reply:
Det er 3 sibling som er flex 25%-50%-25%, hvor høyre får `display: none` når det blir litt smalere, da er gjør `flex-grow` på midten at den fyller ut resten av plassen. Også er det et mellomsteg hvor filleren blir flexa 10%.

Det kan gjøres med margin men da må man håndtere det dynamiske selv med media queries. :thinking: Tror det er mer komplekst enn å bruke en tom div å ta plassen med? :smile: Du må gjerne pulle og ekprimentere litt, ev. så kan vi se på det sammen.